src/main/kotlin/com/cognifide/gradle/aem/pkg/tasks/Deploy.kt

```diff
@@ -129,6 +132,13 @@ open class Deploy : Sync() {
         aem.notifier.notify("Package deployed", "${packages.fileNames} on ${instances.names}")
     }
 
+    private fun tailTaskNotification() {
```

Comment:
this is wrong, project having task `Deploy` could not have task `Tail` (different tasks applied)

Reply:
we need just to check if somewhere (at least in one /other project) there is applied `InstancePlugin`